### PR TITLE
chore(devtool): build_kernel: fix info message

### PR DESCRIPTION
**Pull request by kalyazin**
## Changes

Fix the path in the info message.

Was:
`Kernel binary placed in build/kernel/vmlinux-4.14-x86_64.bin`

Now:
`Kernel binary placed in build/kernel/linux-4.14/vmlinux-4.14-x86_64.bin`

## Reason

The info message says:
```Kernel binary placed in build/kernel/vmlinux-4.14-x86_64.bin```

However the binary is actually located at:
```build/kernel/linux-4.14/vmlinux-4.14-x86_64.bin```

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

~~- [ ] If a specific issue led to this PR, this PR closes the issue.~~
- [x] The description of changes is clear and encompassing.
~~- [ ] Any required documentation changes (code and docs) are included in this PR.~~
~~- [ ] API changes follow the [Runbook for Firecracker API changes][2].~~
~~- [ ] User-facing changes are mentioned in `CHANGELOG.md`.~~
- [x] All added/changed functionality is tested.
~~- [ ] New `TODO`s link to an issue.~~
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [x] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: resources/tests/build_kernel.sh [hunk 1]
```diff
@@ -90,7 +90,9 @@ build_kernel() {
     cp $binary_path $binary_name
 
     popd >/dev/null
-    echo "Kernel binary placed in build/kernel/$binary_name"
+
+    local full_binary_name="$KERNEL_DEST_DIR/$binary_name"
+    echo "Kernel binary placed in ${full_binary_name##$FC_ROOT_DIR/}"
```
Comment (zulinx86) [inline resources/tests/build_kernel.sh:95]: How about preserving the original behavior?
```
diff --git a/resources/tests/build_kernel.sh b/resources/tests/build_kernel.sh
index 42b3deac..2c592a90 100755
--- a/resources/tests/build_kernel.sh
+++ b/resources/tests/build_kernel.sh
@@ -86,10 +86,11 @@ build_kernel() {
 
     make olddefconfig
     make -j $NPROC $target
-    local binary_name="vmlinux-$KERNEL_VERSION-$arch.bin"
-    cp $binary_path $binary_name
 
     popd >/dev/null
+
+    local binary_name="vmlinux-$KERNEL_VERSION-$arch.bin"
+    cp "$KERNEL_DEST_DIR/$binary_path" "$FC_KERNEL_DIR/$binary_name"
     echo "Kernel binary placed in build/kernel/$binary_name"
 }
```

File: resources/tests/build_kernel.sh [hunk 2]
```diff
@@ -90,6 +90,9 @@ build_kernel() {
     cp $binary_path $binary_name
```
Comment (zulinx86) [inline resources/tests/build_kernel.sh:91]: nit: this part is no longer needed.

Comment (kalyazin) [reply resources/tests/build_kernel.sh]: thanks, fixed

**Review by zulinx86 — APPROVED** [review]

File: resources/tests/build_kernel.sh (hunk 1 above)
Comment (zulinx86) [reply resources/tests/build_kernel.sh]: I'm really sorry. I was wrong. The original behavior was to save the kernel under `build/kernel/linux-<version>/`.
https://github.com/firecracker-microvm/firecracker/commit/fb0f950eb2ff72d5bacb1f73c6aadabd733d6a56

Your initial change preserves the original behavior.

**Review by zulinx86 — APPROVED** [review]

File: resources/tests/build_kernel.sh [hunk 3]
```diff
@@ -90,7 +90,9 @@ build_kernel() {
...
 
     popd >/dev/null
-    echo "Kernel binary placed in build/kernel/$binary_name"
+
+    local full_binary_name="$KERNEL_DEST_DIR/$binary_name"
```
Comment (mattschlebusch) [inline resources/tests/build_kernel.sh:94]: super-nitpick: `full_binary_name` -> `full_binary_path`

**Review by mattschlebusch — DISMISSED** [review]

File: resources/tests/build_kernel.sh (hunk 3 above)
Comment (kalyazin) [reply resources/tests/build_kernel.sh]: You're right, this looks more consistent. Fixed, thanks.

**Review by zulinx86 — APPROVED** [review]

**Pull request merged**